test/PerformanceTests/ServiceTests/AssemblyInfo.cs

```diff
@@ -1,3 +0,0 @@
-﻿using Xunit;
-
-[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
I assume we no longer have any tests in this project using Xunit, no?

Reply:
We don't. Still using xunit for some assertions in the ServiceTests, but not for running the tests.